streaming.md

```diff
@@ -142,11 +144,12 @@ message OutputFrame {
 
 ### OutputFrame
 `OutputFrames` represent data coming out of the function. Upon reception of a result data structure on the j-th output stream of a function, an invoker
-1. MUST serialize the data structure to a series of bytes (hereafter the `OutputFrame payload`) by selecting a MIME type that is compatible with the preference ordered list of MIME types specified in the j-th string of `expectedContentTypes` seen in the [`StartFrame`](#StartFrame). As a consequence, the invoker MUST not send any `OutputFrames` until it has received the `StartFrame`. 
-2. If it cannot successfully serialize data using such a MIME type, then it MUST fail this rpc invocation
-3. Otherwise, it MUST set the value of the `payload` field of the frame to the byte representation of the data, and the `contentType` field to the *concrete* MIME type that was selected for serialization
-4. MUST set the `resultIndex` field of the frame to `j`
-5. MAY set additional custom `headers`.
+1. MUST serialize the data structure to a series of bytes (hereafter the `OutputFrame payload`) by selecting a MIME type that is compatible with the preference ordered list of MIME types specified in the j-th string of `expectedContentTypes` seen in the [`StartFrame`](#StartFrame). As a consequence, the invoker MUST not send any `OutputFrames` until it has received the `StartFrame`.
+2. If such a MIME type is not supported, then it MUST fail this rpc invocation with a crafted status error of code [InvalidArgument](https://github.com/grpc/grpc/blob/2a7191abe717d3ce33b5fcbfd6bd118367499cde/doc/statuscodes.md#status-codes-and-their-use-in-grpc) and the error message MUST start with `"Invoker: Not Acceptable"`. An invoker MAY append error details to that message prefix. These details SHOULD start with punctuation to separate them from the error message prefix.
+3. If it cannot otherwise successfully serialize data using such a MIME type, then it MUST fail this rpc invocation
```

Comment:
Should a particular gRPC status code be used?

Reply:
Maybe `UNKNOWN`? WDYT @ericbottard?

Reply:
Yes, UNKNOWN is the default and seems sensible for this case

Reply:
done